users.json

```diff
@@ -279,7 +279,8 @@
         "pongad",
         "poy",
         "shollyman",
-        "tbpg"
+        "tbpg",
+        "odeke-em"
```

Comment:
The names are in sorted order for easier viewing.  I think that you also need to add him to the "yoshi" team.